src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [{ ...state }];
+
+  for (const obj in actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
Используй цикл for of для массивов